e2e/pnpm_workspace/app/a/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+load("@aspect_rules_js//js:defs.bzl", "js_binary", "js_test")
+load("@npm//:link_js_packages.bzl", "link_js_packages")
+
+link_js_packages()
+
+js_binary(
+    name = "main",
+    args = ["foo"],
+    data = [
+        "@npm//@aspect-test/a",
```

Comment:
we should work a bunch on the error messaging for sad paths like if one of these deps is missing

Reply:
created issue to remeber